params/config.go

```diff
@@ -242,72 +242,72 @@ var (
 	// This configuration is intentionally not using keyed fields to force anyone
 	// adding flags to the config to also have to set these fields.
 	AllEthashProtocolChanges = &ChainConfig{
-		ChainId: big.NewInt(1337),
-		HomesteadBlock: big.NewInt(0),
-		DAOForkBlock: nil,
-		DAOForkSupport: false,
-		EIP150Block: big.NewInt(0),
-		EIP150Hash: common.Hash{},
-		EIP155Block: big.NewInt(0),
-		EIP158Block: big.NewInt(0),
-		ByzantiumBlock: big.NewInt(0),
+		ChainId:             big.NewInt(1337),
+		HomesteadBlock:      big.NewInt(0),
+		DAOForkBlock:        nil,
+		DAOForkSupport:      false,
+		EIP150Block:         big.NewInt(0),
+		EIP150Hash:          common.Hash{},
+		EIP155Block:         big.NewInt(0),
+		EIP158Block:         big.NewInt(0),
+		ByzantiumBlock:      big.NewInt(0),
```

Comment:
Hi @gzliudan , do you know how to not auto-format like this?

Reply:
How about golint ?

Reply:
this lint is ok